test/models/taskList/SectionTitleSpec.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.taskList
+
+import models.tasklist.SectionTitle
+import models.tasklist.SectionTitle._
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+import play.api.libs.json.{JsPath, JsSuccess, Json}
+
+class SectionTitleSpec extends AnyFreeSpec with Matchers {
+
+  "EsaTitle" - {
+
+    "must parse to and from json" in {
+      val underTest = Json.toJson(EsaTitle)
+
+      underTest.toString() mustBe s"\"${EsaTitle.toString}\""
```

Comment:
Should be able to remove `.toString`, i.e. `s"\"$EsaTitle\""`

Reply:
Have applied to all the task list related model specs.